### DISCUSSION TITLE
Windows 11 Enterprise Builds Stuck on Version Selection

### DISCUSSION THREAD

**Discussion by Exonical** [vmware/packer-examples-for-vsphere#993] [Help] [answered]
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.

### Project Version

latest

### VMware vSphere

8.0U3

### HashiCorp Packer

1.9.2

### HashiCorp Packer Plugin for VMware vSphere

1.4.0

### Guest Operating System

Windows 11 Enterprise

### Description

When trying to build a Windows 11 Enterprise template, it gets stuck on the selection for theproduct version. This is using the official Windows 11 For Business ISO.

### Expected Behavior

Windows 11 Enterprise version should be auto selected, and install should continue

### Actual Behavior

Virtual machine boots into product selection screen and does not continue without manual intervention

### Steps to Reproduce

Trigger a Windows 11 Enterprise build using Windows 11 Business ISO. Attaching ISO via datastore.

**Comment by tenthirtyam** [comment, marked as answer]
Did you update your variable to pass `vm_inst_os_eval = false`?

By default, the project uses evaluation mode set in the vars. if you set it to `false` it will pass the provided key and version in the xml.

**Comment by Exonical** [comment]
I, in fact, did **NOT** do that. Fixed now! Thanks!

**Comment by tenthirtyam** [comment]
Great to hear!

Also, please note that in [public your repository](https://github.com/Exonical/packer-vmware) whilst you have taken a copy of project and made modification, the removal of the LICENSE would be a violation of the BSD-2 License.

> _Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:_

> _Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer._

**Ryan Johnson**
Distinguished Engineer, VMware by Broadcom
